infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -162,7 +169,7 @@ private void init(LogReplicationMetadataManager metadataManager, LogReplicationC
         this.applyExecutor = Executors.newSingleThreadExecutor(
                 new ThreadFactoryBuilder()
                         .setDaemon(true)
-                        .setNameFormat("snapshotSyncApplyExecutor")
+                        .setNameFormat("snapshotSyncApplyExecutor-" + remoteClusterId)
```

Comment:
Same here regarding the thread name. Just commenting to have a quick reference to the places.